tutorials/basic-topics/sorting/selection-sort.md

```diff
@@ -11,4 +11,112 @@ keywords:
+In order to achieve this, we can use selection sort. 
+
+## Pseudocode
+
+We loop through the array once. <br />
+For each iteration, we keep the index of seperation of the sorted and unsorted subarrays. <br />
+Then, we loop through the array, keeping the index of the minimum element. <br />
+Finally, we move that element to the end of the sorted subarray, and increment the index of seperation. <br />
+We keep doing this until we reach the end of the array, and print out the resulting array. <br />
+
+## Example Run
+
+Let's see the algorithm work on the first test case: `nums = [2, 0, 2, 1, 1, 0]`. This means `n = 6`.
+In the first iteration, we find the minimum value from index 0 to 5, which is 0. This value is moved to the 0th index. <br/>
+The array is now `nums = [0, 2, 2, 1, 1, 0]`. 
+In the second iteration, we find the minimum value from index 1 to 5, which is 0. This value is moved to the 1st index. <br/>
+The array is now `nums = [0, 0, 2, 1, 1, 2]`. 
+In the third iteration, we find the minimum value from index 2 to 5, which is 1. This value is moved to the 2nd index. <br/>
+The array is now `nums = [0, 0, 1, 2, 1, 2]`. 
+In the fourth iteration, we find the minimum value from index 3 to 5, which is 1. This value is moved to the 3th index. <br/>
+The array is now `nums = [0, 0, 1, 1, 2, 2]`. 
+In the fiftt iteration, we find the minimum value from index 4 to 5, which is 2. This value is moved to the 4th index. <br/>
+The array is still `nums = [0, 0, 1, 1, 2, 2]`. 
+In the last iteration, we find the minimum value from index 5 to 5, which is 2. This value is moved to the 5th index. <br/>
+The array is still `nums = [0, 0, 1, 1, 2, 2]`. 
+
+Now, let's look at the solution to the example question!  
+
+<Tabs>
+<TabItem value="cpp" label="C++">
+<SolutionAuthor name="@Bobliuuu"/>
+
+```cpp
+class Solution {
+public:
+    void sortColors(vector<int>& nums) {
+        int n = nums.size(); 
+        // Loop once through the array, keeping the seperation index of the sorted and unsorted subarray
+        for (int i = 0; i < n - 1; i++) {
+            int midx = i;
+            // Loop through array again to find the index of the minimum value in the unsorted subarray
+            for (int j = i + 1; j < n; j++) {
+                // If the current element is smaller than the current minimum element
+                if (nums[j] < nums[midx]) {
+                    // Keep the index of the actual minimum element
+                    midx = j;
+                }
+            }
+            // Swap the minimum value with the current array element (the end of the sorted subarray)
+            swap(nums[i], nums[midx]);
+        }
+    }
+};
+```
+
+</TabItem>
+</Tabs>
+
+### Complexity
+
+The time complexity of this program is $$O(n^2)$$, since we run two for loops to iterate through the array. <br/>
```

Comment:
Why $$O(n^2)$$ - $$ ? 
Just use $O(n^2)$ for both time & space complexity, that's enough.

Remove <br /> tag at last. But a line space between Time and space complexity

Reply:
Run the app locally and attach a reference screenshot or let's discuss it in discord server

Reply:
My previous message was the reference screenshot!
Let's discuss in discord server if you'd like :D